src/modules/Builder/features/ActivityItemsFlow/ItemFlow/SummaryRow/SummaryRow.hooks.ts

```diff
@@ -0,0 +1,21 @@
+import { useState, useEffect } from 'react';
+import { useWatch } from 'react-hook-form';
+
+import { useCurrentActivity } from 'modules/Builder/hooks';
+import { ConditionalLogic } from 'shared/state/Applet';
+
+import { getItemsInUsage } from './SummaryRow.utils';
+
+export const useItemsInUsage = (name: string) => {
+  const { fieldName } = useCurrentActivity();
+  const itemKey = useWatch({ name: `${name}.itemKey` });
+  const conditionalLogic: ConditionalLogic[] = useWatch({ name: `${fieldName}.conditionalLogic` });
```

Comment:
I think these two useWatch calls can be merged into one.
Something like:
const [itemKey, conditionalLogic]: [types] = useWatch({ name: ['name1', 'name2'});